src/edu/csus/ecs/pc2/core/execute/Executable.java

```diff
@@ -212,6 +212,11 @@ public class Executable extends Plugin implements IExecutable {
     private String packageName = "";
 
     private String packagePath = "";
+    
+    //setting this to True will override the prohibition on invoking a Sandbox when running on Windows.
```

Comment:
For items like this consider adding a TODO: to easily identify what should be removed.

Reply:
There is nothing here which "should be removed".   This is a debugging variable intended to remain in the code permanently.   However, I have added a TODO indicating that the variable needs to be changed to False prior to generating a final production/distribution version.